proto/substrait/extensions/extensions.proto

```diff
@@ -72,7 +72,7 @@ message SimpleExtensionDeclaration {
 message AdvancedExtension {
   // An optimization is helpful information that don't influence semantics. May
   // be ignored by a consumer.
-  google.protobuf.Any optimization = 1;
+  repeated google.protobuf.Any optimization = 1;
```

Comment:
meta: I would have liked to pluralize this (i.e. `optimizations`) this, but that is a breaking change due to json encoding.